### PR TITLE
Rate Limiting added for entire website

### PR DESCRIPTION
**Pull request by ATHARVA-GAWAS**

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -88,6 +88,13 @@
 from django.core.exceptions import ValidationError
 from django.http import HttpRequest
 
+from django.views.generic.edit import CreateView
+from django.http import HttpResponse
```
Comment (DonnieBLT) [inline website/views.py:92]: These seem like they can be removed

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Tweaks

**Comment by ATHARVA-GAWAS** [comment]
I have made the suggested changes

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Please remove the remaining unused imports

**Comment by ATHARVA-GAWAS** [comment]
I have made the changes

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Why is this removing the stype code?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
We want this to be site wide. Can you please add it to the middleware? Also please remove any white space changes (changes to lines with no code)

**Pull request closed without merging**